Umbraco-Cloud/Set-Up/Dedicated/index.md

```diff
@@ -0,0 +1,55 @@
+---
+versionFrom: 8.0.0
+versionTo: 10.0.0
+---
+
+# Move your Umbraco Cloud project to dedicated resources
+
+*Dedicated Resources* is a feature on Umbraco Cloud that gives you the option to move your project to a dedicated server. You can choose between a number of *dedicated options* depending on the amount of resources you will need for your project.
+
+In this article, you can read about how you can move your Umbraco Cloud project to dedicated resources and what you need to be aware of before you do so.
+
+## Before you move your project to dedicated resources
+
+Before you decide to move your Umbraco Cloud project, you need to consider a few things:
+
+- Currently, Umbraco Cloud offers dedicated resources for Professional plans only. We expect to include dedicated resources for Starter and Standard plans in the self-service feature in Umbraco Cloud portal later in 2022.
+
+- In case you want to move a project on a Starter or Standard project plan to a dedicated server in Umbraco Cloud, please reach out to our support who will assist you in doing this.
```

Comment:
I'd change this to:

- Projects on Starter and Standard plans can upgrade to Professional to take advantage of Dedicated Resources, or alternatively, contact support for additional options.

Reply:
Aside from the above, it looks really good 👏